### ISSUE TITLE
[tflite] Build tflite using ./build_lib.sh failed!

### ISSUE THREAD

**Issue by ausk** [tensorflow/tensorflow#38028]
<em>Please make sure that this is a build/installation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:build_template</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  Linux16.04 
- TensorFlow installed from (source or binary): source
- TensorFlow version: 2.2
- GCC/Compiler version (if compiling from source):  g++ 5.4

**Describe the problem**

Build tflite using ./build_lib.sh failed!

```sh
tensorflow/lite/tools/make/gen/linux_x86_64/lib/benchmark-lib.a(cpu_backend_context.o): In function `tflite::CpuBackendContext::~CpuBackendContext()':
cpu_backend_context.cc:(.text+0x3e8): undefined reference to `ruy::detail::SystemAlignedFree(void*)'
cpu_backend_context.cc:(.text+0x43d): undefined reference to `ruy::detail::SystemAlignedFree(void*)'
cpu_backend_context.cc:(.text+0x445): undefined reference to `ruy::detail::SystemAlignedAlloc(long)'
cpu_backend_context.cc:(.text+0x470): undefined reference to `ruy::detail::SystemAlignedFree(void*)'
cpu_backend_context.cc:(.text+0x48e): undefined reference to `ruy::detail::SystemAlignedFree(void*)'
cpu_backend_context.cc:(.text+0x4f0): undefined reference to `ruy::detail::SystemAlignedFree(void*)'
cpu_backend_context.cc:(.text+0x4f8): undefined reference to `ruy::detail::SystemAlignedAlloc(long)'
cpu_backend_context.cc:(.text+0x520): undefined reference to `ruy::detail::SystemAlignedFree(void*)'
cpu_backend_context.cc:(.text+0x53e): undefined reference to `ruy::detail::SystemAlignedFree(void*)'
cpu_backend_context.cc:(.text+0x57e): undefined reference to `ruy::ThreadPool::~ThreadPool()'
...
```

As the `ruy` subfolder has been moved into `lite/experimental/ruy`, we should update the `Makefile` to correct the depends.

```diff
tensorflow/lite/tools/make$ git diff ./Makefile
diff --git a/tensorflow/lite/tools/make/Makefile b/tensorflow/lite/tools/make/Makefile
index ef265cc..fa2a3f1 100644
--- a/tensorflow/lite/tools/make/Makefile
+++ b/tensorflow/lite/tools/make/Makefile
@@ -119,7 +119,8 @@ $(wildcard tensorflow/lite/c/*.c) \
 $(wildcard tensorflow/lite/core/*.cc) \
 $(wildcard tensorflow/lite/core/api/*.cc) \
 $(wildcard tensorflow/lite/experimental/resource/*.cc) \
-$(wildcard tensorflow/lite/experimental/ruy/*.cc)
+$(wildcard tensorflow/lite/experimental/ruy/*.cc)  \
+$(wildcard tensorflow/lite/experimental/ruy/ruy/*.cc)
 ifneq ($(BUILD_TYPE),micro)
 CORE_CC_ALL_SRCS += \
 $(wildcard tensorflow/lite/kernels/*.cc) \
@@ -142,14 +143,18 @@ CORE_CC_EXCLUDE_SRCS := \
 $(wildcard tensorflow/lite/*test.cc) \
 $(wildcard tensorflow/lite/*/*test.cc) \
 $(wildcard tensorflow/lite/*/*/benchmark.cc) \
+$(wildcard tensorflow/lite/*/*/*/benchmark.cc) \
 $(wildcard tensorflow/lite/*/*/example*.cc) \
+$(wildcard tensorflow/lite/*/*/*/example*.cc) \
 $(wildcard tensorflow/lite/*/*/test*.cc) \
 $(wildcard tensorflow/lite/*/*/*test.cc) \
 $(wildcard tensorflow/lite/*/*/*tool.cc) \
 $(wildcard tensorflow/lite/*/*/*/*test.cc) \
+$(wildcard tensorflow/lite/experimental/ruy/ruy/test*.cc) \
+$(wildcard tensorflow/lite/experimental/ruy/ruy/profiler/test*.cc) \
 $(wildcard tensorflow/lite/kernels/*test_main.cc) \
 $(wildcard tensorflow/lite/kernels/*test_util*.cc) \
-tensorflow/lite/experimental/ruy/tune_tool.cc \
+tensorflow/lite/experimental/ruy/ruy/tune_tool.cc \
 tensorflow/lite/tflite_with_xnnpack.cc \
 $(MINIMAL_SRCS)

@@ -351,3 +356,4 @@ $(DEPDIR)/%.d: ;
 .PRECIOUS: $(DEPDIR)/%.d

 -include $(patsubst %,$(DEPDIR)/%.d,$(basename $(ALL_SRCS)))
```

And I make a pull request : https://github.com/tensorflow/tensorflow/pull/38029

**Comment by ausk** [comment]
And I add a pull: https://github.com/tensorflow/tensorflow/pull/38029

**Comment by Saduf2019** [comment]
@ausk 
As there is already a PR can we please move this issue to closed as #38029 this would be sufficient to monitor the issue
